tests/console-backend-service/internal/mockice/mockice.go

```diff
@@ -0,0 +1,236 @@
+package mockice
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/dynamic"
+)
+
+func Start(client dynamic.Interface, namespace, name string, port int32) (string, error) {
+	_, err := createConfigMap(client, namespace, name, port)
+	if err != nil {
+		return "", err
+	}
+
+	_, err = createPod(client, namespace, name, port)
+	if err != nil {
+		Stop(client, namespace, name)
+		return "", err
+	}
+
+	_, err = createService(client, namespace, name, port)
+	if err != nil {
+		Stop(client, namespace, name)
+		return "", err
+	}
+
+	return fmt.Sprintf("%s.%s.svc.cluster.local", name, namespace), nil
+}
+
+func Stop(client dynamic.Interface, namespace, name string) {
+	err := deleteResource(client, "configmaps", namespace, name)
+	if err != nil {
```

Comment:
```go
func deleteResourceError(error err, resource, namespace, name string) {
	if err != nil {
		log.Println(fmt.Sprintf("Cannot delete %s %s/%s, because: %v", resource, namespace, name, err))
	}
}
```